src/main/java/uk/gov/hmcts/reform/cpo/auditlog/LoggerAuditRepository.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.cpo.auditlog;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
`@Repository` ?

Reply:
Although the class isn't actually saving anything, it's writing to a log....but I guess you could be treating logs as repository of events....

Its no biggie - you say `Tomato`, I say `Tomato`...

Reply:
Agreed. Resolved in 8fcbda9faaefab2c281ae0d6bc0cee8f12cbcac2.